### PR TITLE
Fix/bad cursor fix

### PR DESCRIPTION
**Pull request by vfrc2**
Fix issue #1880
~~Move editor font to Ace config~~
Update public assets

Consider add node_modules to git ignore
```
diff --git a/.gitignore b/.gitignore
index 92a0a600..8cb7b0f2 100644
--- a/.gitignore
+++ b/.gitignore
@@ -9,3 +9,4 @@ Gemfile.lock
 !.sprockets*
 !lib/gollum/public/gollum/stylesheets/_styles.css
 !.github*
+/node_modules
\ No newline at end of file
```

### CODE REVIEW

File: lib/gollum/public/gollum/javascript/editor/gollum.editor.js [hunk 1]
```diff
@@ -126,6 +126,10 @@
       }
         
       editor.setTheme("ace/theme/tomorrow");
+      editor.setOptions({
+        fontFamaly: 'Consolas, "Liberation Mono", Courier, monospace',
```
Comment (dometto) [inline lib/gollum/public/gollum/javascript/editor/gollum.editor.js:130]: Typo: `fontFamaly`

Comment (vfrc2) [reply lib/gollum/public/gollum/javascript/editor/gollum.editor.js:130]: you right, but it break fix... ))) need to dig more

Comment (dometto) [reply lib/gollum/public/gollum/javascript/editor/gollum.editor.js:130]: Thanks for looking in to it!

**Comment by vfrc2** [comment]
Add Monaco font to font-family, it solves the problem. Seems it caused by bad default font in Chrome on MacOS.

**Comment by dometto** [comment]
> Consider add node_modules to git ignore

Good idea, will do this in #1903!

Thanks for the fix, looks good to me!

**Pull request merged**